packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/DevLauncherPackage.kt

```diff
@@ -7,8 +7,9 @@ import com.facebook.react.uimanager.ViewManager
 import expo.modules.devlauncher.modules.DevLauncherDevMenuExtensions
 import expo.modules.devlauncher.modules.DevLauncherInternalModule
 import expo.modules.devlauncher.modules.DevLauncherModule
+import expo.modules.core.interfaces.Package
 
-class DevLauncherPackage : ReactPackage {
+class DevLauncherPackage : Package, ReactPackage {
```

Comment:
Is `ReactPackage` necessary here?
I think we use `BasePackage` elsewhere 😉

Reply:
It shouldn't be a BasePackage according to this comment: https://github.com/expo/expo/blob/87e36922e5d3c2ae7422a38451dbd7f6379d8aaa/packages/expo-modules-core/android/src/main/java/expo/modules/core/BasePackage.java#L16

And it has to be a `ReactPackage` too for now. This is only the first step of the migration to the `expo-modules`. Right now, I don't want to change native code much. This pull request only changes how the package is linked.